test/attr-for-selected.html

```diff
@@ -158,6 +168,29 @@
         testSelectItem(i);
       });
     });
+
+    suite('default attribute', function() {
+      var selector;
+      var items;
+
+      setup(function () {
+        selector = fixture('defaultAttribute');
+        items = Array.prototype.slice.apply(selector.querySelectorAll('div[some-attr]'));
+      });
+
+      test('setting non-existing value sets default', function() {
+        selector.select('non-existing-value');
+        assert.equal(selector.selected, 'default');
+        assert.equal(selector.selectedItem, items[2]);
+      });
+
+      test('setting non-existing value sets default', function() {
+        selector.multi = true;
+        selector.select(['non-existing-value']);
+        assert.deepEqual(selector.selectedValues, ['default']);
+        assert.deepEqual(selector.selectedItems, [items[2]]);
```

Comment:
I have added a testcase for this, but not yet filtered out the selectedItems that were invalid. The previous implementation of `selectValues` did not filter these either, so therefore I was hesitant to change this behavior. Should I change it anyway?

(Note: `selector.select` only toggles, we need to use `selectedValues` here which is the `multi` variant of `selected` as I later discovered)